packages/lb-annotation/src/core/toolOperation/ScribbleTool.ts

```diff
@@ -332,15 +346,24 @@ class ScribbleTool extends BasicToolOperation {
     }
   }
 
-  public onScribbleEnd() {
-    if (this.startPoint) {
-      this.cacheContext?.closePath();
-      this.cacheContext?.restore();
-      this.startPoint = undefined;
-      this.history.pushHistory(this.cacheCanvasToDataUrl);
+  public onScribbleEnd(e?: MouseEvent) {
+    if (!e) {
+      return;
+    }
+    const originCoordinate = this.getOriginCoordinate(e);
+    const samePoint = isEqual(this.startPoint, originCoordinate);
+    if (!samePoint && !this.lineActive) {
```

Comment:
why？

Reply:
1. To not save the URL of the image to history when clicked but not tagged (same start and end points)

2. The url has been saved in the lineActive state during onScribbleStart, because in this state onMouseUp cannot determine whether the end point is the same as the initial point.

Reply:
why not determine it by lineActive‘s length? Is there any use of start point === end point?

Reply:
The length cannot be used to judge here. The previous point and the current point are used to judge.